# Change declarations of NP_GetMIMEDescription to match one in newer  npapi.h
# See https://bugs.launchpad.net/ubuntu/+source/vlc/+bug/890911
diff -urNad vlc-1.1.13-orig/projects/mozilla/support/npunix.cpp vlc-1.1.13/projects/mozilla/support/npunix.cpp
--- vlc-1.1.13-orig/projects/mozilla/support/npunix.cpp	2011-07-12 21:11:26.000000000 +0300
+++ vlc-1.1.13/projects/mozilla/support/npunix.cpp	2016-08-31 15:24:49.000000000 +0300
@@ -767,7 +767,7 @@
  *  - Netscape uses the return value to identify when an object instance
  *    of this plugin should be created.
  */
-char *
+const char *
 NP_GetMIMEDescription(void)
 {
     return NPP_GetMIMEDescription();
diff -urNad vlc-1.1.13-orig/projects/mozilla/vlcshell.cpp vlc-1.1.13/projects/mozilla/vlcshell.cpp
--- vlc-1.1.13-orig/projects/mozilla/vlcshell.cpp	2011-07-12 21:11:26.000000000 +0300
+++ vlc-1.1.13/projects/mozilla/vlcshell.cpp	2016-08-31 15:23:55.000000000 +0300
@@ -71,7 +71,7 @@
 /******************************************************************************
  * UNIX-only API calls
  *****************************************************************************/
-char * NPP_GetMIMEDescription( void )
+const char * NPP_GetMIMEDescription( void )
 {
     static char mimetype[] = PLUGIN_MIMETYPES;
     return mimetype;
diff -urNad vlc-1.1.13-orig/projects/mozilla/vlcshell.h vlc-1.1.13/projects/mozilla/vlcshell.h
--- vlc-1.1.13-orig/projects/mozilla/vlcshell.h	2011-07-12 21:11:26.000000000 +0300
+++ vlc-1.1.13/projects/mozilla/vlcshell.h	2016-08-31 15:24:19.000000000 +0300
@@ -24,7 +24,7 @@
 #ifndef __VLCSHELL_H__
 #define __VLCSHELL_H__
 
-char * NPP_GetMIMEDescription( void );
+const char * NPP_GetMIMEDescription( void );
 
 NPError NPP_Initialize( void );
 
